Library/Formula/idris.rb

```diff
@@ -1,26 +1,45 @@
-require "formula"
 require "language/haskell"
 
 class Idris < Formula
   include Language::Haskell::Cabal
 
   homepage "http://www.idris-lang.org"
-  url "https://github.com/idris-lang/Idris-dev/archive/v0.9.15.1.tar.gz"
-  sha1 "064608a43f8544b2c15cefde9e6a22a83ebea904"
-  head "https://github.com/idris-lang/Idris-dev.git"
-
-  bottle do
-    sha1 "1fd1c7c3f223512d2869e8790c4e5afb0f196409" => :yosemite
-    sha1 "b8a7ffc5ed4429665658e54cefe400bacbc26c53" => :mavericks
-    sha1 "6dc50e41562eef4d18987f6f673b8a77f22a5512" => :mountain_lion
-  end
 
   depends_on "ghc" => :build
   depends_on "cabal-install" => :build
   depends_on "gmp"
 
+  depends_on "libffi" => :recommended
+  depends_on "pkg-config" => :build if build.with? "libffi"
+
+  stable do
+    url "https://github.com/idris-lang/Idris-dev/archive/v0.9.16.tar.gz"
+    sha1 "01f794c4e516454b8352266c26c92549e90c708f"
+
+    bottle do
+      sha1 "1fd1c7c3f223512d2869e8790c4e5afb0f196409" => :yosemite
+      sha1 "b8a7ffc5ed4429665658e54cefe400bacbc26c53" => :mavericks
+      sha1 "6dc50e41562eef4d18987f6f673b8a77f22a5512" => :mountain_lion
+    end
+  end
+
+  head do
+    url "https://github.com/idris-lang/Idris-dev.git"
+  end
+
   def install
-    install_cabal_package
+    if build.with? "libffi"
+      ENV.prepend_path "PKG_CONFIG_PATH",
+                       "#{Formula["libffi"].opt_prefix}/lib/pkgconfig"
+    end
```

Comment:
Can you remove it and let the test bots run? Manipulation of PKG_CONFIG_PATH for dependencies is normally handled automatically by Homebrew.